app/src/main/kotlin/com/k0d4black/theforce/di/modules/StarWarsApiModule.kt

```diff
@@ -1,47 +1,38 @@
 package com.k0d4black.theforce.di.modules
 
-
-import com.k0d4black.theforce.data.api.HttpClient
-import com.k0d4black.theforce.data.api.LoggingInterceptor
 import com.k0d4black.theforce.data.api.StarWarsApiService
-import dagger.Module
-import dagger.Provides
 import okhttp3.OkHttpClient
 import okhttp3.logging.HttpLoggingInterceptor
+import org.koin.dsl.module
 import retrofit2.Retrofit
 import retrofit2.converter.moshi.MoshiConverterFactory
-import javax.inject.Named
-import javax.inject.Singleton
-
-@Module
-open class StarWarsApiModule {
-
-    @Provides
-    open fun provideLoggingInterceptor(): HttpLoggingInterceptor = LoggingInterceptor.create()
-
-    @Provides
-    open fun provideOkHttpClient(httpLoggingInterceptor: HttpLoggingInterceptor): OkHttpClient {
-        return HttpClient.setupOkhttpClient(httpLoggingInterceptor)
-    }
-
-    @Singleton
-    @Provides
-    open fun provideStarWarsApi(retrofit: Retrofit): StarWarsApiService {
-        return retrofit.create(StarWarsApiService::class.java)
-    }
-
-    @Singleton
-    @Provides
-    open fun provideRetrofit(okHttpClient: OkHttpClient, @Named("baseUrl") baseUrl: String): Retrofit {
-        return Retrofit.Builder()
-            .baseUrl(baseUrl)
-            .client(okHttpClient)
-            .addConverterFactory(MoshiConverterFactory.create())
-            .build()
-    }
-
-    @Provides
-    @Named("baseUrl")
-    open fun provideBaseUrl(): String = "https://swapi.dev/api/"
-
-}
\ No newline at end of file
+import java.util.concurrent.TimeUnit
+
+val NetworkModule = module {
+
+    single { provideService(get()) }
+
+    single { provideRetrofit(get(), provideBaseUrl()) }
+
+    single { provideOkHttpClient() }
+}
+
+fun provideOkHttpClient(): OkHttpClient {
+    val httpLoggingInterceptor = HttpLoggingInterceptor()
+    httpLoggingInterceptor.level = HttpLoggingInterceptor.Level.BODY
+    return OkHttpClient.Builder()
+        .connectTimeout(60L, TimeUnit.SECONDS)
+        .readTimeout(60L, TimeUnit.SECONDS)
+        .addInterceptor(httpLoggingInterceptor).build()
+}
+
+fun provideRetrofit(okHttpClient: OkHttpClient, url: String): Retrofit {
+    return Retrofit.Builder()
+        .baseUrl(url)
+        .client(okHttpClient)
+        .addConverterFactory(MoshiConverterFactory.create()).build()
+}
+
+fun provideService(retrofit: Retrofit): StarWarsApiService = retrofit.create(StarWarsApiService::class.java)
+
+fun provideBaseUrl(): String = "https://swapi.dev/api/"
```

Comment:
This seems to be a duplicate of the network module in the ```di``` module. It can be removed.